src/languages/en.js

```diff
@@ -1295,4 +1296,7 @@ export default {
         chatUserDisplayNames: 'Chat user display names',
         scrollToNewestMessages: 'Scroll to newest messages',
     },
+    threads: {
+        deletedMessage: '[Deleted Message]',
```

Comment:
NAB: maybe key `parentActionDeletedMessage` so its more verbose when this should be used?

Reply:
the typical pattern is to describe the message itself rather than the usage.

would this be better:

```
parentAction: {
     deletedMessage: ...
}
```

Reply:
sure thats fine with me.

Reply:
developers choice 🙂